src/video.py

```
@@ -148,10 +148,12 @@ async def get_video(videoloc, mode, sorted_filelist=False):
     filelist = []
     videoloc = os.path.abspath(videoloc)
     if os.path.isdir(videoloc):
-        globlist = glob.glob1(videoloc, "*.mkv") + glob.glob1(videoloc, "*.mp4") + glob.glob1(videoloc, "*.ts")
+        # breaking the original way to handle files beacause we need dirs/subdirs, be carreful when uploading other than TOS
+        # full directory torrent creation is handled in tracker file, we need this only to get metas.
```

Ruff (0.14.10)

Comment contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF003)